src/gui/HelpWidget.cpp

```diff
@@ -45,6 +45,10 @@ HelpWidget::HelpWidget() : QVBoxWidget() {
     intro->setTextInteractionFlags(Qt::TextBrowserInteraction);
     intro->setOpenExternalLinks(true);
     intro->setMargin(80);
+    intro->setOpenExternalLinks(false);
+
+    // Connect the link clicked signal to a slot
+    connect(intro, SIGNAL(linkActivated(QString)), this, SLOT(onLinkClicked(QString)));
```

Comment:
You can write like this `connect(intro, &QLabel::linkActivated, this, &HelpWidget::onLinkClicked);`. I am suggesting like this because it's this style is everywhere in code. So maintain same too.